experimental/origin-dapp2/src/pages/growth/Welcome.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component, Fragment } from 'react'
+import withEnrolmentModal from './WithEnrolmentModal'
+import Link from 'components/Link'
+
+function InfographicsBox(props) {
+  const { image, title, text } = props
+
+  return (
+    <Fragment>
+      <div className="col infographics d-flex flex-column m-3">
+        <img className="pt-4 mt-auto" src={image} />
+        <div className="text-center title pt-3 pl-3 pr-3">{title}</div>
+        <div className="text-center text p-3 pb-4">{text}</div>
+      </div>
+    </Fragment>
+  )
+}
+
+class GrowthWelcome extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {}
+
+    this.EnrollButton = withEnrolmentModal('button')
+  }
+
+  onSignUp(setOpenedModal) {
+    setOpenedModal({
+      variables: {
+        modalName: 'GrowthEnroll'
+      }
+    })
+  }
+
+  render() {
+    return (
+      <div className="container growth-welcome">
+        <div className="row">
+          <div className="col-6 d-flex flex-column">
+            <Link to="/">
+              <img className="logo" src="/images/origin-logo-footer.svg" />
+            </Link>
+            <div className="title-text">
+              Your friend Aure has invited you to earn <b>FREE Origin Tokens</b>
+            </div>
+            <div className="sub-title-text">
+              Create an account on Origin today and start completing tasks for
+              the chance to will up to 2000 OGN currently valued at 2000 USD.
```

Comment:
nit: type. "the chance to earn up".

I assume this copy will get updated by Matt ? OGN is not valued at $1

Reply:
thanks, fixing. Yup the copys are still to be completed